lib/geolookup/usa/areacodes.rb

```diff
@@ -111,6 +111,80 @@ def state_abbrev
         def county_name
           County.code_to_name(state_code, county_code)
         end
+
+        private
+
+        POP_LOOKUP = {
```

Comment:
If this is a temporary solution, can we have a comment/`TODO` indicating as much?